termd-build-driver/pom.xml

```diff
@@ -61,9 +61,8 @@
     </dependency>
 
     <dependency>
-      <groupId>org.jboss.spec.javax.websocket</groupId>
-      <artifactId>jboss-websocket-api_1.0_spec</artifactId>
-      <scope>provided</scope>
+      <groupId>io.undertow</groupId>
+      <artifactId>undertow-websockets-jsr</artifactId>
```

Comment:
I think we should use library provided by the container. This way we package it with our WAR/EAR.